### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by FrankMugagga**
### Activities

> Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
> Created a directory for your reusable components: /components.
> Installed [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
> [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
> Added 2 routes in your application, each should render a page component:
> /
> /categories

**Created components inside of the /components directory:**

> Contains a list to render individual books
> Contains a form to create new books
> Contains a navigation to link to the 2 routes created

### CODE REVIEW

File: src/Components/Books.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Form from './Form';
+import BookList from './BookList';
+import './Books.css';
+
+function Books() {
+  return (
+    <div className="booksMain">
+      <BookList />
```
Comment (Qoosim) [inline src/Components/Books.js]: - [ ] It is required to pass data prop to this BookList component and use it to render title and author in its stand-alone.
e.g
```
   <BookList book={book}  />
```

- [OPTIONAL] The proper name for this component should been `BookList` component and name the component you call  inside here `Book`.

File: src/Components/BookList.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            UPDATE PROGRESS
+          </buuton>
+        </div>
+
+      </li>
+    </ul>
+  );
+}
+
+export default BookList;
```
Comment (Qoosim) [inline src/Components/BookList.js]: - [ ] This is where to use the book props you pass to the BookList component inside the Book component as child will be used. You will need to pass it as props here too. You can as well destructure it as parameters in the component. Take a look.

```
  const BookList = ({ book }) => {
      const { title, author } = book
       .... the rest of the code here
   }
```
You then use the title and author wherever you need them.

- [OPTIONAL] The proper name for this component should been `Book` component and name the component you call this one inside `BookList`.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @FrankMugagga, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @FrankMugagga , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- App works as expected ✔️ 
- No linter errors ✔️ 
- All requested changes implemented ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**